torch/_dynamo/variables/builder.py

```diff
@@ -932,7 +935,9 @@ def _clone_input(value):
                 example_value, tx=tx, **kwargs
             )
 
-    if isinstance(example_value, torch.Tensor):
+    if isinstance(example_value, torch.Tensor) or (
```

Comment:
I'm a bit confused here, can you check the following lines?
https://github.com/pytorch/pytorch/blob/fe05266fda4f908130dea7cbac37e9264c0429a2/torch/_dynamo/variables/builder.py#L403-L412

If the variable is ```numpy.ndarray```, it has been wrapped as ```NumpyVariable```. I think we should gate with ```config.trace_numpy```. If it's true, we always use ```NumpyTensorVariable```, otherwise use ```NumpyVariable```.

Reply:
Hmm I think `NumpyVariable` is wrapping `numpy` module, where `NumpyTensorVariable` is for `numpy.ndarray`? Similar to `TorchVariable` and `TensorVariable`.

Reply:
Actually no, check these lines:
https://github.com/pytorch/pytorch/blob/fe05266fda4f908130dea7cbac37e9264c0429a2/torch/_dynamo/allowed_functions.py#L277-L281